agent.go

```diff
@@ -368,7 +373,10 @@ func (s *sandbox) readStdio(cid, execID string, length int, stdout bool) ([]byte
 
 	var file *os.File
 	if proc.termMaster != nil {
-		file = proc.termMaster
+		file, err = proc.epoller.run()
```

Comment:
IMO it worth a comment here to state that when `run()` returns, either the pipe is closed or the terminal is readable. And if `file` returned here is `sockR` (a.k.a process termination), `readStdio` returns EOF.

Reply:
Has added some comments here.